pkg/mev/send_bundle.go

```diff
@@ -0,0 +1,175 @@
+package mev
+
+import (
+	"bytes"
+	"context"
+	"crypto/ecdsa"
+	"encoding/json"
+	"fmt"
+	"io"
+	"net/http"
+	"strconv"
+
+	"github.com/ethereum/go-ethereum/accounts"
+	"github.com/ethereum/go-ethereum/common/hexutil"
+	"github.com/ethereum/go-ethereum/core/types"
+	"github.com/ethereum/go-ethereum/crypto"
+)
+
+const (
+	SendBundleID              = 1
+	SendBundleBloxrouteMethod = "blxr_simulate_bundle"
```

Comment:
Simulate bundle for bloxroute?

Reply:
updated in https://github.com/KyberNetwork/tradinglib/pull/16/commits/08da4637ab7ada9a8e2a90d894ae52df70b76723